src/ekobox/operators_card.py

```diff
@@ -46,40 +46,39 @@ def gen_op_card(Q2grid, update=None, name=None):
     serialized = sql.serialize(def_op)
     def_op["hash"] = (sql.add_hash(serialized))[-1]
     if name is not None:
-        export_op_card(name, def_op)
+        dump(name, def_op)
     return def_op
 
 
-def export_op_card(name, op):
-    """Export the operators card in the current directory
+def dump(name, op):
+    """
+    Export the operators card in the current directory
 
     Parameters
     ----------
-    name : str
-        name of the op. card to export
-
-    op : dict
-        op card
-
+        name : str
+            name of the operators card to export
+        op : dict
+            operators card
```

Comment:
Once more: no indentation, no newline at the beginning.

I'm not going to update all at once all the docstrings, but do not revert the correct ones.